src/Tribe/iCal.php

```diff
@@ -270,9 +270,13 @@ public function do_ical_template() {
 	 * @return string
 	 */
 	public function generate_ical_feed( $post = null, $echo = true ) {
-		$this->post = $post;
+		if ( is_numeric( $post ) ) {
```

Comment:
The docblock for this is incorrect - as $post can be an array<int> as well (see line 255 above)